src/ui-components/FileField/FileField.tsx

```diff
@@ -11,15 +11,22 @@ export interface ITextFieldProps {
   id?: string
   inputRef?: RefObject<HTMLInputElement>
   handleFileChange?: (event: React.ChangeEvent<HTMLInputElement>) => void
+  multiple?: boolean
 }
 
-const FileField: FC<ITextFieldProps> = ({ fieldName, inputRef, handleFileChange }) => {
+const FileField: FC<ITextFieldProps> = ({ fieldName, inputRef, handleFileChange, multiple }) => {
   const { field, fieldState } = useController({ name: fieldName })
+  const { append } = useFieldArray({ name: fieldName })
 
   const handelSetFile = (event: React.ChangeEvent<HTMLInputElement>) => {
     if (handleFileChange) {
-      handleFileChange(event)
-      field.onChange(event.target.files && event.target.files?.[0])
+      if (multiple) {
+        handleFileChange(event)
+        append(event.target.files && event.target.files[0])
```

Comment:
if it's multiple why are we saving only first file?

Reply:
done